sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/FileStreamSink.scala

```diff
@@ -47,7 +47,7 @@ class FileStreamSink(
   private val fileLog = new FileStreamSinkLog(sqlContext, logPath.toUri.toString)
   private val fs = basePath.getFileSystem(sqlContext.sparkContext.hadoopConfiguration)
 
-  override def addBatch(batchId: Long, data: DataFrame): Unit = {
+  override def addData(batchId: Long, data: DataFrame): Unit = {
```

Comment:
We don't need to change this name.  Its still a batch of data and one of the parameters is still named `batch`.